packages/core/src/main.ts

```diff
@@ -240,7 +240,7 @@ export async function createHoprNode(
     // Dump entire database to a file if given by the env variable
     const dump_file = process.env.DB_DUMP ?? ''
     if (dump_file.length > 0) {
-      levelDb.dump(dump_file)
+      await levelDb.dump(dump_file)
```

Comment:
Should the dump the async though?

Reply:
The current DB backend in JS unfortunately does not seem support non-async iteration over entries